### PR TITLE
Added Python script to generate Google Pickle String

### PR DESCRIPTION
**Pull request by sreedharande**
Fixes #

## Proposed Changes

  - Generate Google Pickle String
  -
  -

### CODE REVIEW

File: Solutions/GitHub/Hunting Queries/Repository Permission Switched to Public.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+id: a6e2afd3-559c-4e88-a693-39c1f6789ef1
+name: GitHub Repo switched from private to public
+description: |
+  'This hunting query identifies GitHub activites where a repo was changed from private to public that may be a sign of compromise.'
+requiredDataConnectors: []
```
Comment (v-admahe) [inline Solutions/GitHub/Hunting Queries/Repository Permission Switched to Public.yaml:5]: Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

File: Solutions/GitHub/Hunting Queries/Suspicious Fork Activity.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+id: 467e6a70-adc9-49b7-8cf3-f21927c71159
+name: GitHub OAuth App Restrictions Disabled
+description: |
+  'This hunting query identifies a fork activity against a repository done by a user who is not the owner of the repo nor a contributes.'
+requiredDataConnectors: []
```
Comment (v-admahe) [inline Solutions/GitHub/Hunting Queries/Suspicious Fork Activity.yaml:5]: Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

File: Solutions/GitHub/Hunting Queries/Unusual Number of Repository Clones.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+id: ccef3c74-4b4f-445b-8109-06d38687e4a4
+name: GitHub Repo Clone - Time Series Anomly
+description: |
+  'Attacker can exfiltrate data from you GitHub repository after gaining access to it by performing clone action. This hunting queries allows you to track the clones activities for each of your repositories. The visualization allow you to quickly identify anomalies/excessive clone, to further investigate repo access & permissions'
+requiredDataConnectors: []
```
Comment (v-admahe) [inline Solutions/GitHub/Hunting Queries/Unusual Number of Repository Clones.yaml:5]: Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

File: Solutions/GitHub/Hunting Queries/User First Time Repository Delete Activity.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+id: c3237d88-fdc4-4dee-8b90-118ded2c507c
+name: GitHub First Time Repo Delete
+description: |
+  'This hunting query identifies GitHub activites its the first time a user deleted a repo that may be a sign of compromise.'
+requiredDataConnectors: []
```
Comment (v-admahe) [inline Solutions/GitHub/Hunting Queries/User First Time Repository Delete Activity.yaml:5]: Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

File: Solutions/GitHub/Hunting Queries/User Grant Access and Grants Other Access.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+id: f18c4dfb-4fa6-4a9d-9bd3-f7569d1d685a
+name: GitHub User Grants Access and Other User Grants Access
+description: |
+  'This hunting query identifies Accounts in GitHub that have granted access to another account which then grants access to yet another account that may be a sign of compromise.'
+requiredDataConnectors: []
```
Comment (v-admahe) [inline Solutions/GitHub/Hunting Queries/User Grant Access and Grants Other Access.yaml:5]: Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

**Review by v-admahe — CHANGES_REQUESTED** [review]

**Comment by v-admahe** [comment]
@sreedharande Please include sample data for GitHub_CL and  GitHubRepoLogs_CL in the json format in PR.

Thanks,
Aditi Maheshwari

File: Solutions/GitHub/Analytic Rules/GitHub Activities from Infrequent Country.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
+id: f041e01d-840d-43da-95c8-4188f6cef546
+name: GitHub Activites from a New Country
+description: |
+  'Detect activities from a location that was not recently or was never visited by the user or by any user in your organization.'
+severity: Medium
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Analytic Rules/GitHub Activities from Infrequent Country.yaml]: Added

File: Solutions/GitHub/Analytic Rules/Security Vulnerability in Repo.yaml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+id: 5436f471-b03d-41cb-b333-65891f887c43
+name: GitHub Security Vulnerability in Repository
+description: |
+  'This alerts when there is a new security vulnerability in a GitHub repository.'
+severity: Informational
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Analytic Rules/Security Vulnerability in Repo.yaml]: Updated to Low

File: Solutions/GitHub/Analytic Rules/Security Vulnerability in Repo.yaml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Analytic Rules/Security Vulnerability in Repo.yaml]: Added

File: Solutions/GitHub/Analytic Rules/Threat Intel Matches to GitHub Audit Logs.yaml [hunk 9]
```diff
@@ -0,0 +1,46 @@
+id: aac495a9-feb1-446d-b08e-a1164a539452
+name: TI map IP entity to GitHub_CL
+description: |
+  'Identifies a match in GitHub_CL table from any IP IOC from TI'
+severity: Medium
+requiredDataConnectors:
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Analytic Rules/Threat Intel Matches to GitHub Audit Logs.yaml:6]: Added

File: Solutions/GitHub/Analytic Rules/Two Factor Authentication Disabled.yaml [hunk 10]
```diff
@@ -0,0 +1,30 @@
+id: 3ff0fffb-d963-40c0-b235-3404f915add7
+name: GitHub Two Factor Auth Disable
+description: |
+  'Two-factor authentication is a process where a user is prompted during the sign-in process for an additional form of identification, such as to enter a code on their cellphone or to provide a fingerprint scan. Two factor authentication reduces the risk of account takeover. Attacker will want to disable such security tools in order to go undetected. '
+severity: Medium
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Analytic Rules/Two Factor Authentication Disabled.yaml]: Added

File: Solutions/GitHub/Hunting Queries/First Time User Invite and Add Member to Org.yaml [hunk 11]
```diff
@@ -0,0 +1,26 @@
+id: c3237d88-fdc4-4dee-8b90-118ded2c507c
+name: GitHub First Time Invite Member and Add Member to Repo
+description: |
+  'This hunting query identifies a user that add/invite a member to the organization for the first time. This technique can be leveraged by attackers to add stealth account access to the organization.'
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/First Time User Invite and Add Member to Org.yaml]: Added

File: Solutions/GitHub/Hunting Queries/Inactive or New Account Usage.yaml [hunk 12]
```diff
@@ -0,0 +1,46 @@
+id: b8508e24-47a6-4f8e-9066-3cc937197e7f
+name: GitHub Inactive or New Account Access or Usage
+description: |
+  'This hunting query identifies Accounts that are new or inactive and have accessed or used GitHub that may be a sign of compromise.'
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Inactive or New Account Usage.yaml]: Added

File: Solutions/GitHub/Hunting Queries/Mass Deletion of Repositories .yaml [hunk 13]
```diff
@@ -0,0 +1,37 @@
+id: 67da5c4e-49f2-476d-96ff-2dbe4b855a48
+name: GitHub Mass Deletion of repos or projects
+description: |
+  'This hunting query identifies GitHub activites where there are a large number of deletions that may be a sign of compromise.'
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Mass Deletion of Repositories .yaml]: Added

File: Solutions/GitHub/Hunting Queries/Oauth App Restrictions Disabled.yaml [hunk 14]
```diff
@@ -0,0 +1,16 @@
+id: 667e6a70-adc9-49b7-9cf3-f21927c71959
+name: GitHub OAuth App Restrictions Disabled
+description: |
+  'This hunting query identifies GitHub OAuth Apps that have restrictions disabled that may be a sign of compromise. Attacker will want to disable such security tools in order to go undetected. '
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Oauth App Restrictions Disabled.yaml]: Added

File: Solutions/GitHub/Hunting Queries/Org Repositories Default Permissions Change.yaml [hunk 15]
```diff
@@ -0,0 +1,16 @@
+id: ec986fb7-34ed-4528-a5f3-a496e61d8860
+name: GitHub Update Permissions
+description: |
+  'This hunting query identifies GitHub activites where permissions are updated that may be a sign of compromise.'
+requiredDataConnectors: []
```
[the remark this answers is not in the document]
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Org Repositories Default Permissions Change.yaml]: Added

File: Solutions/GitHub/Hunting Queries/Repository Permission Switched to Public.yaml (hunk 1 above)
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Repository Permission Switched to Public.yaml]: Added

File: Solutions/GitHub/Hunting Queries/Suspicious Fork Activity.yaml (hunk 2 above)
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Suspicious Fork Activity.yaml]: Added

File: Solutions/GitHub/Hunting Queries/Unusual Number of Repository Clones.yaml (hunk 3 above)
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/Unusual Number of Repository Clones.yaml]: Added

File: Solutions/GitHub/Hunting Queries/User First Time Repository Delete Activity.yaml (hunk 4 above)
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/User First Time Repository Delete Activity.yaml]: Added

File: Solutions/GitHub/Hunting Queries/User Grant Access and Grants Other Access.yaml (hunk 5 above)
Comment (sreedharande) [reply Solutions/GitHub/Hunting Queries/User Grant Access and Grants Other Access.yaml]: Added

**Comment by v-jayakal** [comment]
@sreedharande , as this is community contribution, as per the policy, we will not have community contributions under solutions. They has to be under individual folders (Connectors in Sentine\Data Connectors, Analytic rules in Sentinel\Detections etc). Please move them out of solutions.

**Pull request closed without merging**